src/components/SUE/report/SueReportLocation.tsx

```diff
@@ -92,13 +102,37 @@ export const SueReportLocation = ({
         locations={locations}
         mapCenterPosition={mapCenterPosition}
         mapStyle={styles.map}
+        isMyLocationButtonVisible
+        onMyLocationButtonPress={() =>
+          Alert.alert(texts.sue.report.alerts.hint, texts.sue.report.alerts.myLocation, [
+            {
+              text: texts.sue.report.alerts.no,
+              onPress: () => null
```

Comment:
Do we need an onPress that returns null?

Reply:
please try without it

Reply:
I tried it and there was no change. I didn't notice any difference between it being `null` or not having `onPress`.